lib/core/src/model.rs

```diff
@@ -1459,6 +1469,44 @@ pub struct LnUrlInfo {
     pub lnurl_withdraw_endpoint: Option<String>,
 }
 
+/// Configuration for asset metadata. Each asset metadata item represents an entry in the
+/// [Liquid Asset Registry](https://docs.liquid.net/docs/blockstream-liquid-asset-registry).
+/// An example Liquid Asset in the registry would be [Tether USD](https://assets.blockstream.info/ce091c998b83c78bb71a632313ba3760f1763d9cfcffae02258ffa9865a37bd2.json>).
+#[derive(Debug, Clone, Serialize)]
+pub struct AssetMetadata {
+    /// The asset id of the registered asset
+    pub asset_id: String,
+    /// The name of the asset
+    pub name: String,
+    /// The ticker of the asset
+    pub ticker: String,
+    /// The precision used to display the asset amount.
+    /// For example, precision of 2 shifts the decimal 2 places left from the satoshi amount.
+    pub precision: u32,
+}
+
+impl AssetMetadata {
+    pub fn to_sat(&self, amount: f64) -> u64 {
+        (amount * (10_u64.pow(self.precision) as f64)) as u64
+    }
+
+    pub fn from_sat(&self, amount_sat: u64) -> f64 {
+        amount_sat as f64 / (10_u64.pow(self.precision) as f64)
+    }
+}
```

Comment:
Nit: this is a bit unconventional. Maybe we could have an `AssetAmount` struct with these methods.

Reply:
@dangeross I think here it would help if at least the methods were renamed to `amount_to_sat` and `amount_from_sat`

Reply:
A method named `to_sat` on a certain struct leads me to think the struct itself will be converted to sat.

Reply:
Ok, I renamed the functions [eab5359](https://github.com/breez/breez-sdk-liquid/pull/659/commits/eab5359db8019228bd98cd534f505892de817a42)